src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs

```diff
@@ -667,6 +691,11 @@ private void RemoveStoredDraftApprenticeshipState()
             TempData.Remove(nameof(AddDraftApprenticeshipViewModel));
         }
 
+        private AddDraftApprenticeshipViewModel PeekStoredDraftApprenticeshipState()
+        {
+            return TempData.GetButDontRemove<AddDraftApprenticeshipViewModel>(nameof(AddDraftApprenticeshipViewModel));
+        }
```

Comment:
Move to `OverlappingTrainingDateRequestController.cs` after merge from master

Reply:
Done